src/schema/teiid.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * This file is part of ciboard-server
+
+ * Copyright (c) 2022 Andrei Stepanov <astepano@redhat.com>
+ * 
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import _ from 'lodash';
+import debug from 'debug';
+import {
+  graphql,
+  GraphQLFieldConfig,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql';
+import { getTeiidClient } from '../services/teiid';
+
+const log = debug('osci:root_query_type');
```

Comment:
Should this be adjusted for the module name?
```suggestion
const log = debug('osci:teiid');
```